packages/web-app-app-store/src/views/AppList.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div class="app-list oc-mb-m">
+    <h2 class="oc-mt-rm" v-text="$gettext('App Store')" />
```

Comment:
I'd suggest adding a link to the install guides for apps here? I naively thought I'd install them directly into my instance and was a bit surprised when it "just" downloaded a zip file 😅

Reply:
Phase 1 is `discovery`, phase 2 is `easy install`, we need backend features for phase 2.

Do you have a good idea where to put such a link? It would be helpful indeed...

Reply:
I guess an ℹ️  icon next to the heading with a tooltip that provides a link to https://owncloud.dev/services/web/#loading-applications would suffice for now?